From c829ce50edd9999a2f0d4a010dc54af8f3a99c9a Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Mon, 27 Jul 2015 15:04:57 +0100
Subject: [PATCH 1/5] Fix typo in option --ignore-cpuid-check

There is a typo in the option, fix it:

   --ingore-cpuid-check  -> --ignore-cpuid-check

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 src/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.cpp b/src/main.cpp
index d692cf3..ac02f79 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -170,7 +170,7 @@ int main(int argc, char *argv[]) {
 					"Enable Dbus."), NULL }, { "exclusive-control", 0, 0,
 			G_OPTION_ARG_NONE, &exclusive_control, N_(
 					"Take over thermal control from kernel thermal driver."),
-			NULL }, { "ingore-cpuid-check", 0, 0, G_OPTION_ARG_NONE,
+			NULL }, { "ignore-cpuid-check", 0, 0, G_OPTION_ARG_NONE,
 			&ignore_cpuid_check, N_("Ignore CPU ID check."), NULL },
 
 	{ NULL } };
-- 
2.4.6

